core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -4794,6 +4841,9 @@ components:
             - ContinueWhilePossible
         externalEntityInfo:
           $ref: '#/components/schemas/ExternalEntityInfo'
+        userComment:
+          type: string
+          description: Comment for the submission
```

Comment:
Should the maximum length for the field be mentioned? It appears that it is 1000-- how was that value selected?

Reply:
This length restriction was mentioned in one of the tickets that was refined previously.